_posts/2020-04-29-azure-ad-authentication-cypress.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: Azure AD Authentication in Cypress Tests
+date: 2020-04-29
+tags: azure ad activedirectory cypress authentication adal react
+author: Tim Veletta
+image: img/blog/azure-ad-cypress/header.png
+---
+
+[Cypress](https://cypress.io) is a browser-based, end-to-end testing framework which makes testing easy and, dare I say it, fun. At Mechanical Rock, we are using Cypress to automate the testing of user flows and interactions with our applications to reduce feedback cycles and prevent issues from affecting users. It has been useful for developers since Cypress tests are relatively easy to write and run as well as our clients since we can automate testing that would have been done manually otherwise.
```

Comment:
I would seperate:

> It has been useful for developers since Cypress tests are relatively easy to write and run as well as our clients since we can automate testing that would have been done manually otherwise.

Into two sentences:

> It has been useful for developers since Cypress tests are relatively easy to write and run. Our clients have also benefited as we are able to automate testing that would have been done manually otherwise.